ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -10,13 +10,17 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+import 'react-grid-layout/css/styles.css';
+import 'react-resizable/css/styles.css';
```

Comment:
I suspect, but don't know, that this might be an issue for library/embed use cases. I'm interested to see what the library build output looks like cause the ability to load CSS like this typically comes from an app's webpack build. We may need to think about how to do this in a way that still works with Dashboards the library.

Reply:
follow up ticket - https://app.shortcut.com/chronosphere/story/29863/convert-react-grid-layout-css-files-to-using-mui-global-styles